src/core/audiosource.cpp

```diff
@@ -244,6 +263,14 @@ void FFMS_AudioSource::ResampleAndCache(CacheIterator pos) {
 		OutPlanes, new_req, DecodeFrame->nb_samples,
 		DecodeFrame->extended_data, DecodeFrame->nb_samples * av_get_bytes_per_sample(CodecContext->sample_fmt), DecodeFrame->nb_samples);
 #else
+#if defined(WITH_SWRESAMPLE)
```

Comment:
The macros stuff is already accepted, but the MSVC linkage is still a matter of discussion here :)
I am sure you can compile ffms2 with MSVC and the official package of ffmpeg. That just works because they provide GCC based .a files in their dev-package and that seems to link sucessfully with MSVC. Actually, I would never have expected this behaviour from Microsoft.
When you compile everything on you own using MSVC first ffmpeg and then ffms2 you will have to handle lib files.